code/datums/gamemode/role/cultist.dm

```diff
@@ -14,17 +16,25 @@
 	if(!.)
 		return
 
+	update_cult_hud()
 	antag.current.add_language(LANGUAGE_CULT)
-	if(!(locate(/spell/cult) in antag.current.spell_list))
+
+	if(ishuman(antag.current) && !(locate(/spell/cult) in antag.current.spell_list))
 		antag.current.add_spell(new /spell/cult/trace_rune, "cult_spell_ready", /obj/abstract/screen/movable/spell_master/bloodcult)
 		antag.current.add_spell(new /spell/cult/erase_rune, "cult_spell_ready", /obj/abstract/screen/movable/spell_master/bloodcult)
 
 /datum/role/cultist/RemoveFromRole(var/datum/mind/M)
```

Comment:
I thought this was named `Drop`
This is the proc I use for removing thralls as an example